MANIFEST.in

```diff
@@ -1,6 +1,21 @@
 include *.md
+include *.rst
 include LICENSE.txt
 include NOTICE
 include pyproject.toml
 recursive-include src/snowflake/connector py.typed *.py *.pyx
+recursive-include src/snowflake/connector/vendored LICENSE*
+
 recursive-include src/snowflake/connector/cpp *.cpp *.hpp
+exclude src/snowflake/connector/arrow_iterator.cpp
```

Comment:
I think it's important for me to point out that while working on this I found that we include the trans-piled `arrow_iterator.cpp` (originally from `arrow_iterator.pyx` by Cython) in our previous source distributions.
From now on the `cpp` file won't be included and when uses install our `sdist` they'll run Cython (under the hood, thanks to PEP-517) themselves to generate it.